test/timer.js

```diff
@@ -105,5 +106,28 @@ module.exports = function runTimerTestSuite() {
         });
       });
     });
+
+
+    it('should record "real" time of function call', function () {
+      var statsd = new StatsD({mock:true});
+      var instrumented = statsd.timer(sleep(100), 'blah');
+
+      instrumented();
+
+      var timeFromStatLine = statsd.mockBuffer[0].match(/blah:(\d+\.\d+)\|/)[1];
+
+      assert.ok(timeFromStatLine >= 100);
+      assert.ok(timeFromStatLine < 120);
```

Comment:
I worry about this becoming a jittery test with this.  How about "< 200" just in case?

Reply:
Yeah, the upper bound was simply to sanity check really so ill up the limit :)